src/WebBridge.ts

```diff
@@ -108,6 +114,26 @@ function selectDevice(): Promise<USBDevice> {
     });
 }
 
+/***
+ * Check is connection with device successful.
+ */
+function checkConnection(): void {
+  if (
+    additionalInfo.text().length &&
```

Comment:
Using tag content length as successful connection flag is not good. I'd prefer to have some global status `CONNECTION_STATUS` and based on its value change message for user.
@yarsanich what do you think?

Reply:
@uraniun resolved